community/libmad/build

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh -e
+
+for patch in *.patch; do
```

Comment:
Change this entire block to:

```sh
patch -p1 < remove-fforce-mem.patch
```

Reply:
Actually, you can remove the patch in its entirety and use:

```sh
sed -i'' '/ -fforce-mem/d' configure
```

Reply:
good call! fixed :)